volume/drivers/nfs/nfs.go

```diff
@@ -101,10 +106,10 @@ func Init(params map[string]string) (volume.VolumeDriver, error) {
 	//mount each nfs server
 	for _, v := range inst.nfsServers {
 		// If src is already mounted at dest, leave it be.
-		mountExists, err := mounter.Exists(src, nfsMountPath+v)
-		if !mountExists {
-			// Mount the nfs server locally on a unique path.
-			syscall.Unmount(nfsMountPath+v, 0)
+
+		// ignore the error and let it retry
+		o, _ := exec.Command("/bin/mount", "-t", "nfs").CombinedOutput()
```

Comment:
good question. i'll see if i can change it.

Reply:
Ok, now I remember (I wrote this code in 2018). pkg.mount.Exists() only checks memory, not the system. And it was failing when I restarted the NFS driver because it thought it was not mounted. By running the mount command, I was able to parse the output and determine if the system thought it was mounted or not. Maybe this code can go into pkg mount?

Reply:
@adityadani and I talked and he showed how to do this using the pkg/mount library. I have updated the code to use that.